### DISCUSSION TITLE
Can not generate CHANGELOG.md file

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [lerna/lerna-changelog#600] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/77786952/142809328-dedddab3-5248-4996-b9ed-a0dc8ab488ff.png)
While I use lerna-changelog, it dose not generate CHANGELOG.md . There are only some git response files.
![image](https://user-images.githubusercontent.com/77786952/142809470-c55f0f0a-6f58-4803-ac2b-63af02dec003.png)

**Comment by Turbo87** [comment, marked as answer]
None of [your pull requests](https://github.com/Linya-IronMan/MyElement/pulls?q=is%3Apr+is%3Aclosed) have any labels, which is why they don't show up

**Comment by JuanRojasBC** [comment]
I am not getting any changelog.md file generated when I run npx lerna-changelog am I doing something wrong

**Reply by Turbo87** [reply]
it is not generating a file automatically. you need to copy or pipe the console output to the file if you want that.
